ui/src/app/pages/profile/profile-registration-step-status.tsx

```diff
@@ -56,42 +59,79 @@ const ProfileRegistrationStepStatus: React.FunctionComponent<Props> =
       incompleteButtonText,
       completedButtonText,
       completeStep,
+      completionTimestamp,
       children
     } = props;
 
-    return (
-      <div style={styles.container}>
-        <div style={styles.title}>
-          { title }
+    if (environment.enableProfileCapsFeatures) {
+      return (
+        <div style={styles.container}>
+          <div style={styles.title}>
+            { title }
+          </div>
+          <FlexRow>
+            <div>
+              { isComplete ? (
+                <FlexRow>
+                  <ClrIcon style={{color: colors.success, width: '2rem', height: '2rem'}}
+                           shape='check-circle' className='is-solid' />
+                  <FlexColumn style={{marginLeft: '0.5rem'}}>
+                    <div>{ wasBypassed ? 'Bypassed By Admin on:' : completedButtonText + ' on:' }
```

Comment:
Add a space after on: In case of eraCommons there is information about username too which is not looking right 

<img width="537" alt="Screen Shot 2019-09-16 at 5 13 51 PM" src="https://user-images.githubusercontent.com/34481816/64994009-7f646480-d8a5-11e9-803c-a25061897752.png">

Reply:
nice catch! with the fix it looks like this:

<img width="543" alt="Screen Shot 2019-09-16 at 11 24 58 PM" src="https://user-images.githubusercontent.com/5375000/65009191-e69c0c00-d8d9-11e9-9619-08cfd7b1b740.png">

Reply:
When none are complete, it looks like this:

<img width="498" alt="Screen Shot 2019-09-17 at 12 37 56 AM" src="https://user-images.githubusercontent.com/5375000/65012144-2ec02c00-d8e4-11e9-82bb-30efc6611294.png">